BookAPITests/NewBookAdditionTests.cs

```diff
@@ -1,76 +1,117 @@
 using System.ComponentModel.DataAnnotations;
 using AutoFixture;
 using BookAPI;
+using Unity;
 using Xunit;
 
 namespace BookAPITests
 {
     public class NewBookAdditionTests
     {
+        IUnityContainer container = new UnityContainer();
```

Comment:
Keeping globals in fixtures in not preferred.
May be return the container from initialize?